test/intl402/PluralRules/default-options-object-prototype.js

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2017 Igalia, S. L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-initializepluralrules
+description: >
+  Monkey-patching Object.prototype does not change the default
+  options for PluralRules as a null prototype is used.
+info: >
+  InitializePluralRules ( collator, locales, options )
+
+    1. If _options_ is *undefined*, then
+      1. Let _options_ be ObjectCreate(*null*).
+---*/
+
+Object.prototype.type = "ordinal";
+let pluralRules = new Intl.PluralRules("en");
+assert.sameValue(pluralRules.resolvedOptions().type, "cardinal");
```

Comment:
The resolveOptions still return an object which [prototype is %ObjectPrototype%](https://github.com/tc39/proposal-intl-plural-rules/commit/1abe8af440fb47d888ace701e8a46559bdbe7b06#diff-adc4f706d081d14e0e322335e03c7727R280).

I think it's valuable to check this is still effective, so one can benefit of this inheritance for the returned value (using toString, etc).

One other good thing for this test is: GetOption should not trigger any observable behaving on the Object.prototype for this new created `options`. You can verify it this using a `Object.defineProperty(Object.prototype, "type", { get() { throw new Test262Error() } } );`. This way you also assert the proper [`GetOption(options, type, ...`](https://github.com/tc39/proposal-intl-plural-rules/commit/1abe8af440fb47d888ace701e8a46559bdbe7b06#diff-adc4f706d081d14e0e322335e03c7727R20) operation.